package.json

```diff
@@ -1,74 +1,71 @@
 {
-  "name": "mpc-phase2-suite",
-  "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
-  "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
-  "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
-  "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
-  "license": "MIT",
-  "private": true,
-  "workspaces": {
-    "packages": [
-      "phase2cli",
-      "firebase",
-      "website"
+    "name": "zkmpc",
+    "description": "MPC Phase 2 suite of tools for conducting zkSNARKs trusted setup ceremonies",
+    "repository": "git@github.com:quadratic-funding/mpc-phase2-suite.git",
+    "homepage": "https://github.com/quadratic-funding/mpc-phase2-suite",
+    "bugs": "https://github.com/quadratic-funding/mpc-phase2-suite/issues",
+    "license": "MIT",
+    "private": true,
+    "keywords": [
+        "typescript",
+        "zero-knowledge",
+        "zk-snarks",
+        "phase-2",
+        "trusted-setup",
+        "ceremony",
+        "snarkjs",
+        "circom"
     ],
-    "nohoist": [
-      "**/snarkjs",
-      "**/snarkjs/**"
+    "scripts": {
+        "build": "lerna run build",
+        "pretest": "yarn build",
```

Comment:
I noticed that running the `yarn test` command is causing a rebuild of all packages, even when there are no changes. This is taking up unnecessary time, could you please look into changing this behaviour?

Reply:
This happens due to the `yarn pretest` script we have discussed above :)

Reply:
We can configure everything to run only where changes happens. Tracked here #224